crates/context/config/src/client.rs

```diff
@@ -1,12 +1,12 @@
-#[macro_use]
-mod transport_macro;
+mod macros;
 use std::borrow::Cow;
 use std::fmt::Debug;
 use std::ops::Deref;
 use std::str::FromStr;
 
 use alloy::signers::local::PrivateKeySigner;
 use eyre::Context;
+use macros::transport;
```

Comment:
move to the block L19-22

https://github.com/calimero-network/core/blob/master/STYLE.md#organize-imports-according-to-the-stdexternalcrate-pattern